src/ncp/spinel.h

```diff
@@ -633,6 +633,24 @@ typedef enum
     SPINEL_PROP_JAM_DETECT_HISTORY_BITMAP
                                         = SPINEL_PROP_PHY_EXT__BEGIN + 5,
 
+    /// Total Tx Time
+    /** Fromat: `L` (read-only)
+     *  Units: ms
+     *
+     * This property returns the totaly amount of time the radio has been
+     * in Tx mode.
+     */
+    SPINEL_PROP_TX_TOTAL_TIME           = SPINEL_PROP_PHY_EXT__BEGIN + 6,
+
+    /// Total Rx Time
+    /** Fromat: `L` (read-only)
+     *  Units: ms
+     *
+     * This property returns the totaly amount of time the radio has been
+     * in Rx mode.
+     */
+    SPINEL_PROP_RX_TOTAL_TIME           = SPINEL_PROP_PHY_EXT__BEGIN + 7,
```

Comment:
Grouping these two properties into a single spinel query seems appropriate here.  Because these properties will not be queried individually and the efficiency of getting both properties in a single call has value, combining these into one call seems like the right course of action.